src/go/rdctl/pkg/factoryreset/factory_reset_windows.go

```diff
@@ -1,59 +1,41 @@
-/*
-Copyright © 2021 SUSE LLC
```

Comment:
We should probably keep some sort of license header?

Reply:
Not sure what happened there.